docs/source/eapol/eapol.rst

```diff
@@ -0,0 +1,69 @@
+===============================================================
+Connecting NI Linux Real-Time Systems to a Wired 802.1X Network
+===============================================================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+
+Introduction
+============
+
+Networks protected by `IEEE 802.1X`_ require that devices authenticate before
+connecting to the network.
+NI Linux Real-Time uses `wpa-supplicant`_ to provide credentials to the network.
+wpa-supplicant supports numerous EAP methods.
+This document provides instructions on how to connect NI Linux Real-Time systems
+to a wired 802.1X network.
+
+.. _IEEE 802.1X: https://en.wikipedia.org/wiki/IEEE_802.1X
+.. _wpa-supplicant: https://w1.fi/wpa_supplicant/
+
+Installation
+============
+
+Support for connecting to wired 802.1X networks is available on the NI Linux RT System
+Image starting with version 25.5.0.
```

Comment:
Style-wise, I think I prefer to use the actual OS version when giving version limits like this, because you can easily interrogate that from the shell with `cat /etc/os-release`. So could you change this to the OS version (and then maybe give the `25.5.0` string in parentheses for completeness)?